css/style.css

```diff
@@ -0,0 +1,65 @@
+@media only screen and (max-width: 1000px){
+
+}
+.welcome-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    justify-content:  center;
+    background-color: #002;
+}
+.nav{
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    background: crimson;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 10;
+}
+p{
+    color: whitesmoke;
+    font-size: 2rem;
+}
+h1{
+    color: white;
+    font-size: 7rem;
+   
+}
+h1,h2{
+    font-family: 'Raleway', sans-serif;
+    font-size: 700;
+    text-align: center;
+}
+
+.nav-list a{
+    display: table;
+    font-size: 2.0rem;
+    padding: 1rem;
+
+}
+a{
+    text-decoration: none;
+    color: white;
+    display: inline;
+}
+ul.nav-list{
+    
```

Comment:
- [ ] Kindly remove this line.

Reply:
Great comment, It would also be nice to explain why this change has to be made.